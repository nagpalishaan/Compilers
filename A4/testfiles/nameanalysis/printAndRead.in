int main() {
    int a = read(a);
    print();
}