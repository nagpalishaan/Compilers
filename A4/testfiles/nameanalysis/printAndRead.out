Error at line 2: expected type does not match. Type: lang.ast.BoolType@37271612 != Expected type: lang.ast.IntType@ed7f8b4
Error at line 2: symbol 'a' is not declared
Error at line 3: expected type does not match. Type: lang.ast.BoolType@37271612 != Expected type: lang.ast.IntType@ed7f8b4
