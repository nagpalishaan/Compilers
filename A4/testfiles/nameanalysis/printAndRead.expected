Error at line 2: symbol 'a' is not declared
